src/beast/doc/websocket.qbk

```diff
@@ -309,9 +309,9 @@ and received a close frame.
 
 During read operations, Beast automatically reads and processes control
 frames. Pings are replied to as soon as possible with a pong, received
-pongs are delivered to the pong callback. The receipt of a close frame
-initiates the WebSocket close procedure, eventually resulting in the error
-code [link beast.ref.websocket__error `error::closed`] being delivered
+ping and pongs are delivered to the ping callback. The receipt of a close
```

Comment:
This sentence reads weird:

> Pings are replied to as soon as possible with a pong, received ping and pongs are delivered to the ping callback.

Why is there a comma after "pong"? Also, if pings are replied to as soon as possible with a pong, why are they also delivered to the ping callback? Or are they only delivered if automatic response to PING is enabled? Or?